### PR TITLE
Yogbot

### PR DESCRIPTION
**Pull request by oisin100**
Adds discord messaging to the following Elements:
- Asay
- OOC
- Tickets
- Round begin

Also adds a C++ DLL to send future POST requests to any website.

:cl:
rscadd: OOC And Asay are now sent to Discord.
rscadd: Admins are now alerted via discord when there are open tickets and no admins online.
rscadd: Round begin are now announced through discord.
/:cl:

### CODE REVIEW

File: tools/ByondPostDLL/ByondDLL.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+	Copyright (C) 2016 Oisin Carr
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
```
Comment (Iamgoofball) [inline tools/ByondPostDLL/ByondDLL.cpp:5]: Nope.

This is AGPLv3 due to being in the codebase. Fix your license.

File: licenses/libcurl-license [hunk 2]
```diff
@@ -0,0 +1,22 @@
+COPYRIGHT AND PERMISSION NOTICE
```
Comment (Iamgoofball) [inline licenses/libcurl-license:1]: Nope, this is AGPLv3 due to the codebase.

Comment (RemieRichards) [inline licenses/libcurl-license:1]: No, this isn't required to for the game to run (or shouldn't be) and as such can be added to the codebase under it's own licensing (so long as it's clearly stated, as it is)

File: tools/ByondPostDLL/ByondDLL.cpp (hunk 1 above)
Comment (george99g) [inline tools/ByondPostDLL/ByondDLL.cpp:5]: https://www.gnu.org/licenses/gpl-faq.en.html#GPLModuleLicense

Pretty sure the GPL is compatible with the AGPL.

Plus, I don't think you can actually even follow the AGPL with this.

**Pull request merged**